packages/augur-sdk/src/state/create-api.ts

```diff
@@ -74,7 +74,7 @@ export function buildSyncStrategies(client:Augur, db:Promise<DB>, provider: Ethe
 
     const endWarpSyncBlockNumber = await warpSyncStrategy.start();
     const staringSyncBlock = Math.max(await (await db).getSyncStartingBlock(), endWarpSyncBlockNumber || uploadBlockNumber);
-    const endBulkSyncBlockNumber = await bulkSyncStrategy.start(staringSyncBlock, currentBlockNumber);
+    const endBulkSyncBlockNumber = await bulkSyncStrategy.start(staringSyncBlock + 1, currentBlockNumber);
```

Comment:
this seems wrong. If the upload block number is used for example this will skip the logs from the initial upload.

Reply:
Love it. Done!